### ISSUE TITLE
[Wiki module design] Azure Resource Reference (ARR) alignment - Parameter names (with what exceptions, e.g., cmk or networking input parameters that we are trying to align across all resources supporting those)

### ISSUE THREAD

**Issue by eriqua** [Azure/ResourceModules#2676]

**Comment by MariusStorhaug** [comment]
Examples of this issue, the implementation of "sku".
![MicrosoftTeams-image](https://user-images.githubusercontent.com/17722253/217877645-ebab36d8-e892-4245-b132-e14b869adad3.png)
![MicrosoftTeams-image (2)](https://user-images.githubusercontent.com/17722253/217877648-aee51617-af24-4d6b-9979-3f05f5da43b3.png)
![MicrosoftTeams-image (1)](https://user-images.githubusercontent.com/17722253/217877655-34e07c3b-0032-42d9-b0e5-f79e8fa7309b.png)

**Comment by AlexanderSehr** [comment]
Duplicate

**Comment by eriqua** [comment]
@AlexanderSehr, if we close this, we must make sure that #2677 also includes this wiki enhancement.

**Comment by rahalan** [comment]
This will be covered by AVM specs in the future.

**Comment by AlexanderSehr** [comment]
Hey @eriqua, do you think we should revisit this in BRM? I presume the original reasoning for this issue are still very much relevant. Yet, if it's not a spec, I guess it's up to the module owner's discretion :o

**Comment by eriqua** [comment]
@AlexanderSehr I agree this is still relevant for AVM. If we don't have a spec already we should. I'd discourage reinventing the naming wheel for anything except for:
- when using *resourceId instead of *Id
- for everything covered by interfaces
- anything else ?
